src/shared/scrapers/US/NY/index.js

```diff
@@ -84,13 +84,16 @@ const scraper = {
   async scraper() {
     this.url = 'https://health.data.ny.gov/api/views/xdss-u53e/rows.csv?accessType=DOWNLOAD';
     this.type = 'csv';
-    const data = await fetch.csv(this.url, false);
+    const data = await fetch.csv(this.url, false, { alwaysRun: true });
 
     const dateField = 'Test Date';
 
     // FIXME when we roll out new TZ support!
-    const fallback = process.env.USE_ISO_DATETIME ? new Date(datetime.now.at('America/New_York')) : datetime.getDate();
+    const fallback = new Date(
+      process.env.USE_ISO_DATETIME ? datetime.today.at('America/New_York') : datetime.getDate()
+    );
     let scrapeDate = process.env.SCRAPE_DATE ? new Date(`${process.env.SCRAPE_DATE} 12:00:00`) : fallback;
+    scrapeDate.setDate(scrapeDate.getDate() - 1);
```

Comment:
Confused as to why this is necessary, can you explain?

Reply:
Because the most recent data is always a day behind, the old way makes it look 2 days in a row had the same data. This new way pushes all the data back by one day instead of just the most recent one.

Reply:
![image](https://user-images.githubusercontent.com/32887801/79093640-e73b3a80-7d22-11ea-96aa-2af2063312b4.png)


See how the last two dates are the same

Reply:
Mmm yeah the last two days are the same, but then tomorrow gets updated and it rises normally. Seems fine... @qgolsteyn @camjc @shaperilio thoughts?

Reply:
Feels like ideally we’d not have a number when we don’t know it yet?

Reply:
I think we'll have to always put the latest known number as the actual number. If they don't update over the weekend, then yes it'll be a flat part.

Reply:
If the table has dates, then we know exactly the day the data represents. So we should not be manipulating anything, there should be no duplicating the last point because our scrape date is "in the future". There should simply be no data for April 12 if the table only has up to April 11.

It would then be the job of the front end (e.g. the map) to know that on April 12, it should show April 11 data (and notify the user on hover/click that this is the last update date). We are dealing with multiple governments un multiple timezones so no one is updating all at the same time.

Reply:
I agree with @shaperilio and @camjc - we mustn't make assumptions about the data if we don't have it.  Since we already have a standard for reporting missing data (use `undefined`), let's do that.

Am doing so in https://github.com/covidatlas/coronadatascraper/pull/974.